data/json/items/resources/misc.json

```diff
@@ -37,33 +37,17 @@
     "id": "down_mattress",
     "type": "GENERIC",
     "category": "spare_parts",
-    "name": "down_mattress",
-    "name_plural": "down_mattresses",
+
+    "name": "down mattress",
+    "name_plural": "down mattresses",
     "description": "This is a single, or twin, sized down filled mattress.",
     "weight": 20000,
     "volume": "300000 ml",
     "price": 1000,
     "material": [ "cotton" ],
     "symbol": "0",
     "color": "white",
-    "use_action": { "type": "deploy_furn", "furn_type": "f_down_mattress" }
-  },
-  {
-    "id": "fuse",
```

Comment:
Why is this removed?

Reply:
I was getting the fuse item as an error that needed to be resolved so when I looked in the experimental version I've been playing it looks like fuse had already been removed.  But I could just be looking at too many different versions and getting timeline confused.

Reply:
You should update master branch from upstream and rebase this branch.

Reply:
If this last change fails I think I'm going to close this PR and do what you are saying.  Or can I rebase this branch with the PR open? Will that cause more problems?

Reply:
You don't need to close PR to rebase. And you may check if your local branch is ok after rebasing before pushing changes to Github.